src/development/platform-integration/ios/apple-frameworks.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Leveraging Apple's System Libraries
+description: Learn how to apply knowledge of Apple's APIs and frameworks when building Flutter apps.
+---
+
+This document is for iOS developers looking to apply
+their existing knowledge of Apple's APIs within Flutter.
+This includes accessing the device hardware and interacting 
+with specific frameworks like `HealthKit` or `MapKit`.
+
+<!-- Add once SwiftUI PR is merged -->
+<!-- For an overview of how the SwiftUI framework compares to Flutter, 
+see [Learning Flutter as a SwiftUI developer][].  -->
+
+### Flutter plugin overview
+When developing an app with Flutter, you can interact 
+with system libraries using plugins. 
+Plugins are Dart libraries that contain platform-specific code. 
+In the case of an iOS app, 
+the platform code can be written in Objective-C or Swift. 
+
+In your Dart code, you use the plugin's Dart API and 
+behind the scenes the native plugin code will directly call 
```

Comment:
```suggestion
behind the scenes the native plugin code directly calls
```

Reply:
**question:** Can the native plugin code indirectly call the system library? If not, remove _directly_ because it doesn't add any value.